plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -52,13 +52,12 @@
           siteUri="https://github.com/objectify/objectify/wiki" >
       <libraryDependency id="appengine-api" />
```

Comment:
This dependency should go away.

But then, this `<library id="objectify"` becomes identical to the `<library id="objectify6"` below. So ideally, we may be able to git rid of this entirely.

Reply:
Oh, wait. That's not exactly true. The "group" attribute is different.

In any case, the `appengine-api` dependency should go away.

Reply:
You're right, done.